src/distilabel/steps/generators/huggingface.py

```diff
@@ -243,20 +243,19 @@ def _dataset_info(self) -> Dict[str, DatasetInfo]:
         Returns:
             The dataset information.
         """
-        repo_id = self.repo_id
-        config = self.config
 
         try:
-            return get_dataset_infos(repo_id)
+            return get_dataset_infos(self.repo_id)
         except Exception as e:
             # The previous could fail in case of a internet connection issues.
             # Assuming the dataset is already loaded and we can get the info from the loaded dataset, otherwise it will fail anyway.
+            # TODO: This fails when the step is not loaded using make_generator_step.
```

Comment:
is this something you have to tackle on this PR?

Reply:
It's done, forgot to remove the comment!